specification/storageactions/resource-manager/Microsoft.StorageActions/stable/2023-01-01/storageactions.json

```diff
@@ -460,7 +460,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.StorageTasks/previewActions": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.StorageActions/locations/{location}/previewActions": {
       "post": {
```

Comment:
This URL change was already approved as part of this PR - https://github.com/Azure/azure-rest-api-specs/pull/25716

But It was not merged because this bigger PR had more changes.